src/test/java/net/minecraftforge/debug/gameplay/loot/ShearsLootTests.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.debug.gameplay.loot;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.component.DataComponents;
+import net.minecraft.gametest.framework.GameTest;
+import net.minecraft.gametest.framework.GameTestHelper;
+import net.minecraft.world.entity.EquipmentSlot;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.ShearsItem;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.Blocks;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;
+import net.minecraftforge.gametest.GameTestHolder;
+import net.minecraftforge.registries.DeferredRegister;
+import net.minecraftforge.registries.ForgeRegistries;
+import net.minecraftforge.registries.RegistryObject;
+import net.minecraftforge.test.BaseTestMod;
+
+@Mod(ShearsLootTests.MODID)
+@GameTestHolder("forge." + ShearsLootTests.MODID)
+public class ShearsLootTests extends BaseTestMod {
+    public static final String MODID = "shears_loot";
+
+    private static final DeferredRegister<Item> ITEMS = DeferredRegister.create(ForgeRegistries.ITEMS, MODID);
+    private static final RegistryObject<Item> MODDED_SHEARS = ITEMS.register("modded_shears", ModdedShearsItem::new);
+
+    public ShearsLootTests(FMLJavaModLoadingContext context) {
+        super(context);
+        this.testItem(lookup -> MODDED_SHEARS.get().getDefaultInstance());
+    }
+
+    @GameTest(template = "forge:empty3x3x3")
+    public static void test_shears_drop_blocks(GameTestHelper helper) {
+        Block[] shearsDropBlocks = {
+                //leaves
```

Comment:
This is missing the following blocks that use shears in vanilla:
GLOW_LICHEN, LARGE_FERN, TALL_GRASS, TALL_SEAGRASS

Reply:
sorry, but I cannot figure out how to test these 4 blocks. LARGE_FERN, TALL_GRASS, TALL_SEAGRASS are DoublePlantBlocks and even if I position the UPPER and LOWER half correctly on a dirt block, the destroyBlock method does not drop any item. When I break the test placed blocks by hand, an item is dropping. The same thing for the GLOW_LICHEN, because it seems to have another weird BlockStateBehaviour.
I don't know if it is worth the time.